### PR TITLE
renaming the default parameter for bindings

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

**Comment by soninaren** [comment]
fixes #128

**Comment by soninaren** [comment]
need to update more bindings adding additional commits

**Comment by soninaren** [comment]
updated

**Comment by soninaren** [comment]
@mattchenderson  or @christopheranderson can you review?

File: Bindings/bindings.json [hunk 1]
```diff
@@ -268,7 +268,7 @@
             "type": "serviceBus",
             "displayName": "Azure Service Bus",
             "direction": "out",
-            "defaultParameterName": "myQueueItem",
+            "defaultParameterName": "sbOutputQueueItem",
```
Comment (christopheranderson) [inline Bindings/bindings.json]: outputSbQueueItem would be more consistent

File: Bindings/bindings.json [hunk 2]
```diff
@@ -243,7 +243,7 @@
             "type": "serviceBusTrigger",
             "displayName": "Azure Service Bus",
             "direction": "trigger",
-            "defaultParameterName": "myQueueItem",
+            "defaultParameterName": "sbQueueItem",
```
Comment (christopheranderson) [inline Bindings/bindings.json]: inputSbQueueItem would be more consistent

**Comment by christopheranderson** [comment]
Approved

File: Bindings/bindings.json (hunk 2 above)
Comment (soninaren) [inline Bindings/bindings.json]: following the format for trigger parameter with prefix "my"

**Pull request merged**